ol7/checks/oval/file_permissions_unauthorized_sgid.xml

```diff
@@ -0,0 +1,78 @@
+<def-group>
+  <definition class="compliance" id="file_permissions_unauthorized_sgid" version="2">
+    <metadata>
+      <title>Find setgid files system packages</title>
+      <affected family="unix">
+        <platform>Oracle Linux 7</platform>
```

Comment:
An OVAL check by the same name exists (ref: shared/xccdf/system/permission/files.xml). I'm not really familiar with Oracle Linux. Is something unique about OL7 that the shared content doesn't handle?

Reply:
Shawn, in general Oracle Linux (OL) mostly based on RHEL userspace packages source code. In addition to what is released in RHEL these packages might also contain additional fixes or improvements for Oracle products while some of packages might be excluded from OL distribution as not supported. OL also includes Unbreakable Enterprise Kernel, Ksplice and some other components developed in Oracle. Thus I would say from STIG perspective OL is 95% compatible with RHEL and majority of existing rhel content could be just reused. Though in some cases either guide content or OVAL checks might require platform specific changes.

Speaking of file_permissions_unauthorized_sgid.xml it is just copied from rhel7 content and adopted to OL as part of initial patch. Unfortunately I couldn't get it in shared oval folder despite it is referenced from shared xccdf. Further it will be reviewed and aligned with OL7 STIG, thus minor modifications could be considered. 
In theory it could be shared if 'var_suid_whitelist' which is product specific defined outside of this rule.